Java

```diff
@@ -1,44 +1,55 @@
 import java.util.Scanner; 
-import java.text.NumberFormat;
-class student{   
+    class Student{
+        String name;
+        String id;
+        int hours = 0;
+
+        public Student(String name, String id){
+            this.name = name;
+            this.id = id;
+        }
+
+        public void addHours(double hours){
+            this.hours += hours;
+        }
+
+        public double getHoursAsPercentage(double totalHours){
+            return hours * 100.0 / totalHours;
+        }
+    }
+class Main{   
```

Comment:
If you want to run the class then please know that the name of the file needs to match the name of the main class.